src/starknetCli.ts

```diff
@@ -198,7 +213,7 @@ export async function runStarknetCallOrInvoke(
   let funcName, inputs: string;
   try {
     [funcName, inputs] = await encodeInputs(
-      filePath,
+      solAbiPath ?? '',
```

Comment:
we should try to regenerat the solidity abi if its not found;
we have the path to solidity file

Reply:
The solidity abi file is always generated unless it fails to compile, in which case it returns immediately and it gets not here. The same happens with solAbi, which is being used in the command later.